src/screens/TopicPageScreen/index.js

```diff
@@ -437,7 +443,7 @@ const TopicPageScreen = (props) => {
         refreshAnonymous={onDeleteBlockedPostCompleted}
         screen="topic_screen"
       />
-    </SafeAreaProvider>
+    </View>
   );
 };
-export default withInteractionsManaged(TopicPageScreen);
+export default TopicPageScreen;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The wrapping container has been changed from `SafeAreaProvider` to `View`. Also, the `withInteractionsManaged` HOC has been removed from the export statement. These changes are consistent with the PR summary. However, removing `withInteractionsManaged` might affect the interaction management of this screen. Make sure this change doesn't negatively impact user interactions.
```diff
- </SafeAreaProvider>
+ </View>
- export default withInteractionsManaged(TopicPageScreen);
+ export default TopicPageScreen;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this make the screen need a longer time to load, that's why I removed it